src/teamwork.sh

```diff
@@ -47,11 +47,19 @@ teamwork::pull_request_closed() {
   local -r user=$(github::get_sender_user)
   local -r pr_url=$(github::get_pr_url)
   local -r pr_title=$(github::get_pr_title)
+  local -r pr_merged=$(github::get_pr_merged)
```

Comment:
You need to create this function to fetch this field from the JSON file. This function doesn't exist. 

You can find in the `github.sh` file all the helpers to get information from Github, and by using `jq` library you can easily process the JSON file.

Reply:
oh maaaaan, I hate vscode, always forget to save the file.
I was there hanging unsaved.